### ISSUE TITLE
Treating slider steps with different values

### ISSUE THREAD

**Issue by rivella50** [sghall/react-compound-slider#115]
Hi there,

i'm just wondering how i can create a slider with a step interval which does not always have the same value? For instance a date slider where the single steps should be months. Since not all months have the same amount of days the step value cannot be a certain value of millis if each slider movement would have to match the last day of each month.
Is there a solution for such a scenario or would i have to deal with it by myself (e.g. determine the last day of a month by the received ms in the onUpdate/onChange callbacks) ?

**Comment by RanSolo** [comment]
There's an example of a dateslider in the readme.

**Comment by rivella50** [comment]
I know that example. But here the step interval is always the same (half an hour) which doesn't answer my question.

**Comment by RanSolo** [comment]
Well you were talking about months being different lengths of days, not changing the minute intervals.

**Comment by gitwahid** [comment]
Has anyone figured it out?

**Comment by rivella50** [comment]
Nope, i solved it differently with a step interval of a day (which has of course always the same amount of millis).
@RanSolo Please read again my initial question. I was never talking about minute intervals.
